-- AchievementsData (ModuleScript)

local Achievements = {
	["001"] = {
		Name = "A Step Into Memory",
		Description = "The first of many steps. The journey from a grey world into a world of color begins.",
		IconId = "rbxassetid://115736524648016",
		BadgeId = 3534687389501816,
		Secret = false,
		SortOrder = 1,
	},
	["002"] = {
		Name = "Whispers on the Wind",
		Description = "You've reached the highest point, where the sky feels closer and the view is your reward.",
		IconId = "rbxassetid://128357877675592",
		BadgeId = 1927276057952552,
		Secret = false,
		SortOrder = 2,
	},
	["003"] = {
		Name = "Behind the Scenery",
		Description = "Some places aren't on the main path. Thanks for finding our little hideout.",
		IconId = "rbxassetid://82179356783665",
		BadgeId = 2162491696052003,
		Secret = false,
		SortOrder = 3,
	},
	["004"] = {
		Name = "Forest Bathed in Gold",
		Description = "Lost in the tranquility of the Midway Forest, where sunlight filters through the leaves.",
		IconId = "rbxassetid://113117899419999",
		BadgeId = 4206255053574394,
		Secret = false,
		SortOrder = 4,
	},
	["022"] = {
		Name = "Our Secret Hideout",
		Description = "You've found the treetop hideout where the three brothers' adventures began. A place filled with echoes of laughter and childhood dreams.",
		IconId = "rbxassetid://109407618842768",
		BadgeId = 3589707056257992,
		Secret = true,
		SortOrder = 5,
	},
	["005"] = {
		Name = "An Unseasonal Bloom",
		Description = "A splash of pink in a world of green. A beautiful, hidden sight for those who wander.",
		IconId = "rbxassetid://88232564487204",
		BadgeId = 4015162763232980,
		Secret = false,
		SortOrder = 6,
	},
	["006"] = {
		Name = "The Mountain's Tears",
		Description = "Where the river begins its descent with a thunderous yet peaceful roar. ",
		IconId = "rbxassetid://96832668932925",
		BadgeId = 1587577956585598,
		Secret = false,
		SortOrder = 7,
	},
	["023"] = {
		Name = "Sunken Ship",
		Description = "In the middle of the lake, a forgotten vessel holds memories instead of gold. You've uncovered a relic of past adventures.",
		IconId = "rbxassetid://96100121008599",
		BadgeId = 3800527535132697,
		Secret = true,
		SortOrder = 8,
	},
	["007"] = {
		Name = "A Home in the Woods",
		Description = "You've found the rustic shack where the bonds of brotherhood were forged.",
		IconId = "rbxassetid://127278651162964",
		BadgeId = 1799733677249347,
		Secret = false,
		SortOrder = 9,
	},
	["008"] = {
		Name = "The Distant Peak",
		Description = "You've looked upon the mountain that holds so many memories. A promise of the journey ahead.",
		IconId = "rbxassetid://88099592065220",
		BadgeId = 3766260360256692,
		Secret = false,
		SortOrder = 10,
	},
	["009"] = {
		Name = "An Unbreakable Bond",
		Description = "Three cups, one promise. You've witnessed the heart of the story at the old tree.",
		IconId = "rbxassetid://121523949089023",
		BadgeId = 362055923619741,
		Secret = false,
		SortOrder = 11,
	},
	["010"] = {
		Name = "To Brotherhood!",
		Description = "A silent toast to a bond that transcends time.",
		IconId = "rbxassetid://76419288504460",
		BadgeId = 3986037988325084,
		Secret = true,
		SortOrder = 12,
	},
	["024"] = {
		Name = "Echoes of a Thousand Journeys",
		Description = "You've reached the summit's most sacred place, where the memories of countless travelers are immortalized. Thank you for being a part of our story.",
		IconId = "rbxassetid://137500373484142",
		BadgeId = 4332228681706726,
		Secret = true,
		SortOrder = 13,
	},
	["025"] = {
		Name = "Where the Fire Sleeps",
		Description = "Among the crimson blooms, a promise endures. You've paid your respects at a sacred place where a fiery will rests, but is never forgotten.",
		IconId = "rbxassetid://86485122826542",
		BadgeId = 336951938502611,
		Secret = true,
		SortOrder = 14,
	},
	["026"] = {
		Name = "A Glimpse of the Future",
		Description = "A ship of dreams has anchored in a sea of memories. You've found a vessel that has yet to sail, a sunny promise of adventures to come.",
		IconId = "rbxassetid://125306490772457",
		BadgeId = 650036646564504,
		Secret = true,
		SortOrder = 15,
	},
	["027"] = {
		Name = "The Brothers' Chronicle",
		Description = "Within the pages of a dusty journal, you've witnessed the full story: three cups of sake, a shared vow, and a bond that transcends time itself.",
		IconId = "rbxassetid://103014604005219",
		BadgeId = 2416052435426761,
		Secret = true,
		SortOrder = 16,
	},
	["028"] = {
		Name = "A Message Across the Sea",
		Description = "On the deck of a ship sailing towards the dawn, you've found a confession left by another traveler—a tale of hope, distance, and a promise to meet again. Someone's heartfelt story has become a part of your journey.",
		IconId = "rbxassetid://83410582255471",
		BadgeId = 2100267740907392,
		Secret = true,
		SortOrder = 17,
	},
	["011"] = {
		Name = "Pathfinder",
		Description = "5 journeys complete. The routes are becoming familiar, like a well-loved song.",
		IconId = "rbxassetid://118648161355993",
		BadgeId = 1503561715523843,
		Secret = false,
		SortOrder = 18,
	},
	["012"] = {
		Name = "Dawn Inheritor",
		Description = "10 journeys complete. You carry the will of the dawn with every step you take.",
		IconId = "rbxassetid://98543734736497",
		BadgeId = 3322240052183804,
		Secret = false,
		SortOrder = 19,
	},
	["013"] = {
		Name = "Keeper of Memories",
		Description = "25 journeys complete. Every rock and tree tells you a story, and you've heard them all.",
		IconId = "rbxassetid://106550087651147",
		BadgeId = 3837739145993210,
		Secret = false,
		SortOrder = 20,
	},
	["014"] = {
		Name = "Spirit of the Mountain",
		Description = "50 journeys complete. You no longer walk the path; you are a part of it.",
		IconId = "rbxassetid://94614897280016",
		BadgeId = 4499357404913970,
		Secret = false,
		SortOrder = 21,
	},
	["015"] = {
		Name = "Corvo Legend",
		Description = "100 journeys complete. Your footsteps are etched into the history of this mountain.",
		IconId = "rbxassetid://133507449792066",
		BadgeId = 627117002861419,
		Secret = false,
		SortOrder = 22,
	},
	["016"] = {
		Name = "Eternal Voyager",
		Description = "250 journeys complete. A journey without end. A memory relived, forever.",
		IconId = "rbxassetid://108472306111385",
		BadgeId = 2874943980944227,
		Secret = false,
		SortOrder = 23,
	},
	["017"] = {
		Name = "Breaching the Fourth Wall",
		Description = "You found one of the architects of this world in their natural habitat.",
		IconId = "rbxassetid://113978675934042",
		BadgeId = 264969490543190,
		Secret = true,
		SortOrder = 24,
	},
	["018"] = {
		Name = "An Ascended Escort",
		Description = "One of the architects of this world has given you a lift. Hope you enjoyed the view from up there.",
		IconId = "rbxassetid://133148043918701",
		BadgeId = 2098118086036838,
		Secret = true,
		SortOrder = 25,
	},
	["019"] = {
		Name = "A Founder's Favor",
		Description = "The original creator carried you. You are a very, very special and notable person.",
		IconId = "rbxassetid://136770063231106",
		BadgeId = 996851983020908,
		Secret = true,
		SortOrder = 26,
	},
	["020"] = {
		Name = "Leading the Architects of The World",
		Description = "For once, you get to drag them along for the journey.",
		IconId = "rbxassetid://95198600113179",
		BadgeId = 521309340449283,
		Secret = true,
		SortOrder = 27,
	},
	["021"] = {
		Name = "A Guided Tour",
		Description = "You were personally escorted by a developer... by force or by choice.",
		IconId = "rbxassetid://95198600113179",
		BadgeId = 3589707056257992,
		Secret = true,
		SortOrder = 28,
	},
}

return Achievements
