--!nonstrict
--[[
	TipsData (ModuleScript)
]]

local Tips = {
	"The view from the summit is best at sunset.",
	"A promise was made at the old tree on the cliff. Can you find it?",
	"Help a tired friend by carrying them on your back.",
	"Only the top 4 donators get their avatar featured on wanted posters around the map.",
	"Look for hidden paths in the forest, you might find a shortcut!",
	"Complete more journeys to earn prestigious titles like 'Eternal Voyager' or 'Dawn Inheritor'.",
	"Find a scenic spot? Use an emote and get ready to take a picture!",
	"Want a special perks? Check out the VIP pass in the store!",
	"Reaching Dadan's House is a great milestone on your journey.",
	"Light up the path with a Vintage Lantern, available in the store.",
	"Leave your mark on Mount Corvo! The top 4 supporters are immortalized on wanted posters.",
	"Cause some chaos! Trip up other players with the Lasso, available in the store.",
	"Want to see your face on a wanted poster? Become a top donator on the leaderboard!",
	"Not all titles are earned through journeys. Explore and interact with the world to find secret titles!",
	"There are hidden achievements waiting to be discovered. Try interacting with everything!",
	"Enjoying your journey? Liking (👍) and favoriting (⭐) the game helps a lot!",
	"Join our official Roblox group to unlock an exclusive 'Nakama' title!",
	"Your avatar on a wanted poster? Reach the top of the donation leaderboard to make it happen!",
	"Did you know you can change the color of your Vintage Lantern? Equip it to access the lantern settings!",
	"Want a cleaner view for a screenshot? Look for a button to hide the UI!",
	"Join our Discord server to share screenshots and hang out with the community!",
	"Photo mode has special filters to capture the perfect nostalgic moment.",
	"The three sake cups at The Oath Tree represent an unbreakable bond of brotherhood.",
}

return Tips
